### PR TITLE
Small volume corrections

### PR DESCRIPTION
**Pull request by TomMaullin**
This PR addresses issue #21 by adding the small volume option to the SwE display interface. It also addresses a (horrific) bug causing the display window to not reload correctly when a new option is selected.

### CODE REVIEW

File: swe_results_ui.m [hunk 1]
```diff
@@ -600,10 +600,10 @@
         uicontrol(Finter,'Style','PushButton','String','small volume','FontSize',FS(10),...
             'ToolTipString',['Small Volume Correction - corrected p-values ',...
             'for a small search region'],...
-            'Callback','spm_VOI(SwE,xSwE,hReg);',...
+            'Callback','swe_VOI(SwE,xSwE,hReg);',...
             'Interruptible','on','Enable','on',...
             'Position',[015 095 100 020].*WS,...
-            'Enable','off')
+            'Enable','on')
```
Comment (nicholst) [inline swe_results_ui.m]: Minor point: `'Enable','on'` option is given twice.

**Comment by TomMaullin** [comment]
Hi @nicholst ,

The bug is fixed!! And this PR is now ready for review! Please let me know if everything looks alright to you and, if possible, if everything runs correctly on your machine! Please let me know if you have any feedback/suggestions also!

**Comment by nicholst** [comment]
Awesome! Was there a trick? A key “SPM GUI API lessons learned”?
-- 
_________________________________________________________
Thomas Nichols, PhD
Professor of Neuroimaging Statistics
Nuffield Department of Population Health | University of Oxford
Big Data Institute | Li Ka Shing Centre for Health Information and Discovery
Old Road Campus | Headington | Oxford | OX3 7LF | United Kingdom
T: +44 1865 743590 | E: thomas.nichols@bdi.ox.ac.uk
W: http://nisox.org | http://www.bdi.ox.ac.uk

**Comment by TomMaullin** [comment]
It turns out the problem was to do with the setup methods in `swe_results_ui`. The `"setup"` method of [`swe_results_ui`](https://github.com/NISOx-BDI/SwE-toolbox/blob/master/swe_results_ui.m#L234) is the first method to be called whenever a display is made. The error came from how the `"setup"` method does the following two steps:

 - Step 1: It first sets up the base outline of the GUI. This is done by calling `swe_results_ui` with the `"SetupGUI"` input argument on [line 331](https://github.com/NISOx-BDI/SwE-toolbox/blob/master/swe_results_ui.m#L331).
  - Step 2: It then, on [line 513](https://github.com/NISOx-BDI/SwE-toolbox/blob/master/swe_results_ui.m#L513), makes everything currently displayed in the GUI perminent. It does this by giving all of the child nodes of the GUI window the label `"PermRes"` (short for perminent results). The SPM functions are coded such that if the GUI is reset then the nodes with the label "PermRes" will not change (so for example page formatting etc).

The error here was caused by the table of results was being created in the wrong place. The table was being created in the `"SetupGUI"` method on [line 561](https://github.com/NISOx-BDI/SwE-toolbox/blob/master/swe_results_ui.m#L561) which was called during step 1. It was then being made permanent by step 2 and not being deleted when the display was reset. To overcome this I moved the call to `swe_list` to after step 2 (on [line 517](https://github.com/tommaullin/SwE-toolbox/blob/svc/swe_results_ui.m#L517)).

I guess the SPM GUI API lesson learned here would be that anything made in the "SetupGUI" method is here to stay for good. If you need something to change it shouldn't be created in this method.

**Comment by nicholst** [comment]
Awesome!  Thanks so much!

**Comment by TomMaullin** [comment]
Great - is this okay to merge?

**Comment by nicholst** [comment]
I played around with this and it's great.  Could we, however, offer a warning that *if* WB has been run, no FWE inferences are available.  Something like

> WARNING: No FWE (voxel or cluster) results available for small volumes. Create a new WB analysis with a restricted analysis mask to obtain FWE results on a small volume. 

each time 'small volume' is clicked should avoid confusion.

**Comment by TomMaullin** [comment]
Ah yes apologies! The warning must have slipped my mind, I will do this now!

**Comment by TomMaullin** [comment]
Hi @nicholst ,

This warning should now be added. Please let me know if you have any further feedback/are happy to merge!

**Comment by nicholst** [comment]
Super!  THanks, merge when ready.

**Comment by nicholst** [comment]
Looks great! Fire away.

**Pull request merged**